server/src/main/java/oap/ws/security/server/OrganizationWS.java

```diff
@@ -117,15 +124,16 @@ public void storeUser( @WsParam( from = BODY ) User user, @WsParam( from = PATH
     }
 
     @WsMethod( method = GET, path = "/{oid}/user/{email}" )
-    @WsSecurity( role = Role.ORGANIZATION_ADMIN )
+    @WsSecurity( role = Role.USER )
     public Optional<User> getUser( @WsParam( from = PATH ) String oid,
                                    @WsParam( from = PATH ) String email ) {
         if( organizationStorage.get( oid ).isPresent() ) {
             final Optional<User> userOptional = userStorage.get( email );
             if( userOptional.isPresent() ) {
                 final User user = userOptional.get();
 
-                return user.organizationId.equals( oid ) ? Optional.of( user ) : Optional.empty();
+                return user.role.equals( Role.ADMIN ) || user.organizationId.equals( oid ) ?
```

Comment:
Isn't getUser executed by one of these:
1) Admin
2) Everyone else from this organization (in our case, user or org_admin)